pkg/cloud/services/ec2/instances_test.go

```diff
@@ -5691,3 +6017,59 @@ func TestGetCapacityReservationSpecification(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCapacityBlocksInstanceMarketOptionsRequest(t *testing.T) {
```

Comment:
If we change the function names as suggested above, this will need to change too.

Reply:
Yup, done.